### ISSUE TITLE
Invalid content type in response to server discovery request

### ISSUE THREAD

**Issue by saaj** [StephenOTT/TAXII-Server#39]
Steps to reproduce
------------------
`curl -v --user foo:bar -H "Accept: application/taxii+json;version=2.1" http://localhost:8080/taxii2/`

Expected result
---------------
Response `Content-Type` is `application/taxii+json;version=2.1` which is required by [4.1 Server Discovery][1].

Actual result
-------------
`Content-Type` is `application/stix+json;version=2.1`. 

Neither is the discovery response body a valid STIX 2.1.

[1]: https://docs.oasis-open.org/cti/taxii/v2.1/cs01/taxii-v2.1-cs01.html#_q0a03pfr5x7n

**Comment by StephenOTT** [comment]
@saaj good catch.  Its a bug with the default action/provider not kicking it.  I will post a fix for this today or tomorrow.
